wordle-web/src/router/index.ts

```diff
@@ -0,0 +1,29 @@
+import { createRouter, createWebHistory } from 'vue-router'
+import HomeView from '../views/HomeView.vue'
+
+const router = createRouter({
+  history: createWebHistory(import.meta.env.BASE_URL),
+  routes: [
+    {
+      path: '/',
+      name: 'home',
+      component: HomeView
+    },
+    {
+      path: '/about',
+      name: 'about',
+      // route level code-splitting
+      // this generates a separate chunk (About.[hash].js) for this route
+      // which is lazy-loaded when the route is visited.
+      component: () => import('../views/AboutView.vue')
+    },
+    {
+      path: '/game',
+      name: 'game',
+
+      component: () => import('../views/gameView.vue')
```

Comment:
I think Grant mentioned today in class that it's better to import first and then use the component instead of doing it one line (in the case of same line import, the page game will load when it is needed vs right away). It depends on your case of course but because it isn't resource intensive, I would go ahead and import it first and then use it.

Reply:
I edited this, thanks.